### ISSUE TITLE
Vorlagen umbenennen

### ISSUE THREAD

**Issue by larsik78** [verdigado/sunflower#528]
Es wäre ganz hilfreich, wenn man einmal angelegte Vorlagen auch umbenennen könnte. Und dass man die Vorlagenübersicht auch über das Menü im Dashboard und nicht nur über den Block-Editor erreicht (oder ich bin blind).

**Comment by albig** [comment]
Das ist eher eine WordPress-Frage bzw. man müsste Sunflower in ein Block-Theme umwandeln. Dann könnte man die Vorlagen über Design -> Website-Editor finden.

Aber man kommt auch so ran:
 
![2024-03-19_09-37](https://github.com/verdigado/sunflower/assets/3354500/145edcf5-d692-4d4a-abcf-83f6ec384a26)

Dann landet man in einer Übersicht unter

https://example.com//wp-admin/edit.php?post_type=wp_block

![2024-03-19_09-40](https://github.com/verdigado/sunflower/assets/3354500/c8bff5ea-8766-4854-b754-5453c4c48bbd)

**Comment by larsik78** [comment]
Wie man rankommt, weiß ich ja. War auch eher ein Verbesserungsvorschlag. Wichtig wäre umbenennen.

**Comment by albig** [comment]
Das kannst Du dann ja machen, wenn Du den Eintrag bearbeitest. Das ist ja einfach nur der Titel des "Beitrags".

![2024-03-19_09-50](https://github.com/verdigado/sunflower/assets/3354500/c4613267-d276-44de-a213-428407d2f922)

**Comment by larsik78** [comment]
Stimmt, wieder was gelernt. Auch wenn das im Block-Editor nicht so richtig intuitiv ist - für mich. Aber umso besser.

Dann können wir das hier wieder zu machen. :-)

**Comment by albig** [comment]
Ich halt das mal offen, damit wir das dokumentieren. Gerne Freiwillige vor!

**Comment by alexschnapper** [comment]
Ah, ich kann die Dokumentation dahingehend mal updaten... Nehme ich gerne an mich das Thema. Ticket kann gerne an mich assigned werden.
